bin/rucio

```diff
@@ -1672,6 +1672,29 @@ def download(args):
                      'eventType': 'download',
                      'eventVersion': version.RUCIO_VERSION[0]}
 
+    t_appid = os.environ.get('RUCIO_TRACE_APPID')
+    t_dataset = os.environ.get('RUCIO_TRACE_DATASET')
+    t_datasetscope = os.environ.get('RUCIO_TRACE_DATASETSCOPE')
+    t_eventtype = os.environ.get('RUCIO_TRACE_EVENTTYPE')
+    t_pq = os.environ.get('RUCIO_TRACE_PQ')
+    t_taskid = os.environ.get('RUCIO_TRACE_TASKID')
+    t_usrdn = os.environ.get('RUCIO_TRACE_USRDN')
+
```

Comment:
Getting the env. variable in argparse and propagating them as argument/dictionary to the download function will be cleaner, e.g.:
`parser.add_argument('--appid', default=os.environ.get('RUCIO_TRACE_APPID', None))`

Reply:
if we do it that way, we must hide the options from the user.

..., help=argparse.SUPPRESS, ...

Reply:
that's a really good point. Moving toward a download api that can be called in a multi-threads env. was one of the motivation for the requested change.